### DISCUSSION TITLE
Google Developer Advocate proposal for v8port ecosystem

### DISCUSSION THREAD

**Discussion by netskink** [zopencommunity/meta#846] [General]
Hello

I would like to reach out to Google for a Developer Advocate for use with the V8 port here on ZOSOpenTools.  Here is a slide deck to provide to Google.  Please review and comment.

[Request for Google Developer Advocate 20240730.pdf](https://github.com/user-attachments/files/16443119/Request.for.Google.Developer.Advocate.20240730.pdf)

**Comment by lbdyck** [comment]
this looks interesting to me - only comment is I would suggest a slide that describes the z/OS environment (runs on IBM zSeries hardware), including a short explanation of UNIX System Services, and then spell out USS instead of using the abbreviation.  Then again I'm assuming an audience for this could be Google folks who have no clue about IBM Z.

**Reply by netskink** [reply]
That is a good point.

**Comment by netskink** [comment]
Per Lionel's comment.
[Request for Google Developer Advocate 20240730-2.pdf](https://github.com/user-attachments/files/16444369/Request.for.Google.Developer.Advocate.20240730-2.pdf)

**Comment by lbdyck** [comment]
Suggestion  define terms like mainframe, IBM Z, USS, and z/OS up front.

For slide 6 how about:

The IBM mainframe is a server that runs on the IBM Z series hardware with the latest being the IBM z16. The server is capable of running multiple operating system concurrently, including z/OS, z/VM, and Linux. The z/OS operating system includes an integrated UNIX environment known as z/OS UNIX System Services, referred to as USS, that supports ......

**Comment by netskink** [comment]
Change relative to  Lionels and Rays comments via discord
[Google Developer Advocate public.pdf](https://github.com/user-attachments/files/16444715/Google.Developer.Advocate.public.pdf)

**Comment by catherdersoftware** [comment]
In the section on who has emulators running in the cloud, if AWS is running a z/OS emulation, then I'd mention that, because no one wants to look bad if the other one has it.

From my English/communications hat: make sure things are cased consistently, such as UNIX. (Yeah, I've seen The Open Group get fussy about that.)

**Comment by netskink** [comment]
I don't know if IBM or Amazon has compute engine equivalents running zdt's.  I know AMZN has EC2's and most assuredly can a zdt, i just don't know personally.  I also don't know much about IBM cloud.  Do they have Compute Engines/EC2 equivalents?

**Reply by IgorTodorovskiIBM** [reply]
There's Wasi as a Service via IBM cloud: https://www.ibm.com/cloud/wazi-as-a-service

**Reply by netskink** [reply]
Ahh, cool.  That is something I was unaware of.  Thanks Igor!

**Reply by netskink** [reply]
I wonder if I could experiment with that on my own dime or would be too expensive.

**Comment by netskink** [comment]
[Google Developer Advocate public.pdf](https://github.com/user-attachments/files/16456468/Google.Developer.Advocate.public.pdf)

Has updates for 

- z/OS USS -> z/UNIX
- Reduce wordiness
- Added some info on enterprise growth vs marketshare
- Added small ask vs big ask

**Reply by lbdyck** [reply]
Fix title on slide 6 (USS) :)

**Comment by netskink** [comment]
[Google Developer Advocate public.pdf](https://github.com/user-attachments/files/16456703/Google.Developer.Advocate.public.pdf)

**Comment by MikeFultonDev** [comment]
Lets put the request in terms of the z/OS Open Tools community instead of IBM/Broadcom.
To that end, I'd recommend we drop the last 2 pages listing contacts and, if you are willing John, list the z/OS Open Tools community contact as you as the focal point?
Also - since our IBM public system is not yet 'online', I'd recommend we drop that information and instead say that the community can make a system available (and then John - you can work with IBM and/or Broadcom to make that happen). 

@IgorTodorovskiIBM @gabylb for your comment

**Reply by IgorTodorovskiIBM** [reply]
That sounds reasonable to me

**Reply by netskink** [reply]
fwiw, Per suggested putting names and titles to lend weight.  How about this compromise, remove the names from the slide deck and include them in the email?

**Reply by netskink** [reply]
Also Mike, the slides do have a z/OS Open Tools slant in later versions.  In the latest version here, which slide needs further adjustment?  Regarding your other comments. I'll adjust them in the next revision in a few minutes.
[Google_Developer_Advocate_pf20240801.pdf](https://github.com/user-attachments/files/16460025/Google_Developer_Advocate_pf20240801.pdf)

**Reply by MikeFultonDev** [reply]
i'm ok with the names in the mail. if it's for 'weight' probably just need shereen, mike, per, you

**Reply by netskink** [reply]
sounds good.  I'll upload the doc with your revisions in one moment.  I really wish I could simply do a public share.  So much easier.  Comments and revision tracking FTW.

**Reply by netskink** [reply]
regarding this:

>  if you are willing John, list the z/OS Open Tools community contact as you as the focal point?

Add as contacts, Igor and myself?  Err, is that an 'and' or a true 'as'?

**Comment by netskink** [comment]
[Google_Developer_Advocate_pf20240801.pdf](https://github.com/user-attachments/files/16460315/Google_Developer_Advocate_pf20240801.pdf)
